vcx/libvcx/src/credential_def.rs

```diff
@@ -18,7 +18,7 @@ pub struct CredentialDef {
     tag: String,
     name: String,
     source_id: String,
-    issuer_did: String,
+    issuer_did: Option<String>,
```

Comment:
I'm confused on how this is an optional parameter.  I guess its no different than when we had it originally be an empty string upon creation.  Perhaps in a future iteration we require it as a parameter to create?

Reply:
Yes, this is optional. It is not required in the code and must be optional in order to not break compatibility.